rpm_post_install.sh

```diff
@@ -0,0 +1,3 @@
+mkdir -p /usr/share/openstack-dashboard/openstack_dashboard/local/enabled/
```

Comment:
Report error and prompt user the installation failed (exit 1) when /usr/share/openstack-dashboard not exists(openstack-dashboard is not installed yet.).
mkdir and put f5-xxxxx there is useless, cannot run either.